### ISSUE TITLE
feature request: export names as bloom filter

### ISSUE THREAD

**Issue by Dobatymo** [philipperemy/name-dataset#35]
Since the memory usage is very high, it would be great if names lists could be returned as bloom filters to use with less available memory.

**Comment by philipperemy** [comment]
By bloom filter, you mean we only load the most common names? And lazy load the rest?

**Comment by Dobatymo** [comment]
Since the full loaded name list takes up a lot of memory. I would be nice to export them as a [bloom filter](https://en.wikipedia.org/wiki/Bloom_filter) to reduce memory usage and accept some errors.
But it's probably out of scope for this library. Also it can be implemented by the user.

**Comment by philipperemy** [comment]
Yeah let's say it's out of scope. In 99.9% of the time, it should not be a problem. Thanks for suggesting.
